### ISSUE TITLE
Download link broken

### ISSUE THREAD

**Issue by gadjou** [square/picasso#1948]
Download link on the main web site is broken (error 404)

**Comment by SalvatoreT** [comment]
Yeah, it should redirect to this AAR instead of the JAR which doesn't work.
https://repo1.maven.org/maven2/com/squareup/picasso/picasso/2.71828/picasso-2.71828.aar
